index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <link rel ="stylesheet" href="./master.css"></link>
+    <title>Twitter</title>
+</head>
+<body>
+    <section id="container">
+    <div id="white">
+        <div id="header">
```

Comment:
You can have these inputs on a header tag, instead of adding a id to a div. These inputs should also be in a `<form>` tag.